src/VideoToolbox/VTCompressionSession.cs

```diff
@@ -17,7 +18,13 @@
 using CoreVideo;
 
 namespace VideoToolbox {
+
+#if NET
+	[SupportedOSPlatform ("ios8.0")]
+	[SupportedOSPlatform ("tvos10.2")]
```

Comment:
I wonder if we should make these strings constants?

Reply:
hmm... that change was not meant to be included in _this_ PR but yeah I came to the same ~~conclusion~~ question

If we use strings we'll end up with mistakes - even if I added introspection tests against _most_ typos
OTOH using string constants requires a bit more typing
* we can have sharpie generate them...
* but I'd rather only have the old ones inside binding files (and limit the dual set to manual bindings)

Maybe some internal short hands like `_.ios_8_0`...